Makefile.am

```diff
@@ -5606,8 +5606,10 @@ BUSNAMES_TARGET_WANTS += \
 	org.freedesktop.resolve1.busname
 
 GENERAL_ALIASES += \
-	$(systemunitdir)/systemd-resolved.service $(pkgsysconfdir)/system/multi-user.target.wants/systemd-resolved.service \
-	$(systemunitdir)/systemd-resolved.service $(pkgsysconfdir)/system/dbus-org.freedesktop.resolve1.service
+	$(systemunitdir)/systemd-resolved.service $(pkgsysconfdir)/system/multi-user.target.wants/systemd-resolved.service
+
+SYSTEM_UNIT_ALIASES += \
+	systemd-resolved.service dbus-org.freedesktop.resolve1.service
```

Comment:
This doesn't look right. resolved has an [Install] section, you may enable/disable it. This makefile snippet simply enables it by default, so that "systemctl disable" works correctly.

Or to say this differently: if clients ask for org.freedesktop.resolve1 on the bus, they should not get it unless it is actually enabled.